src/core/TransformControls.tsx

```diff
@@ -32,7 +33,15 @@ export type TransformControlsProps = ReactThreeFiber.Object3DNode<TransformContr
     onObjectChange?: (e?: THREE.Event) => void
   }
 
-export const TransformControls = React.forwardRef<TransformControlsImpl, TransformControlsProps>(
+export const TransformControls = React.forwardRef<TransformControlsImpl, TransformControlsProps>((props, ref) => {
+  const { mode } = props
+  if (mode === 'all') {
```

Comment:
@FarazzShaikh: The current version of `Manipulator3D` only supports all modes, not activating specific modes, from what understand of [the code](https://github.com/sketchpunklabs/manipulator3d).
[Would be great to use an array of active modes in the future.](https://github.com/pmndrs/drei/issues/927#issuecomment-1145622004)

Reply:
Turning off modes isn't something I want to support & maintain right now.

Reply:
Sounds good. I really like the UX you created with all 3 enabled anyway 👍 

Adding an `all` mode or something similar should suffice.